ui/messages/en.json

```diff
@@ -1,18 +1,17 @@
 {
   "common": {
     "brand": "Red Hat",
-    "product": "Streams for Apache Kafka",
-    "title": "Red Hat Streams for Apache Kafka console",
-    "description": "Red Hat Streams for Apache Kafka is a lightweight, high-performance, robust messaging platform.",
+    "product": "streams for Apache Kafka",
+    "title": "Red Hat streams for Apache Kafka console",
+    "description": "Red Hat streams for Apache Kafka is a lightweight, high-performance, robust messaging platform.",
     "search_hint": "Filter by {label}"
   },
   "learning": {
     "links": {
-      "overview": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.5/html/amq_streams_on_openshift_overview",
-      "gettingStarted": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.5/html/getting_started_with_amq_streams_on_openshift",
-      "connecting": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.5/html/developing_kafka_client_applications",
-      "topicOperatorUse": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.5/html/deploying_and_managing_amq_streams_on_openshift/using-the-topic-operator-str#doc-wrapper"
-
+      "overview": "https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html/streams_for_apache_kafka_on_openshift_overview",
+      "gettingStarted": "https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html/getting_started_with_streams_for_apache_kafka_on_openshift",
+      "connecting": "https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html/developing_kafka_client_applications",
+      "topicOperatorUse": "https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html/deploying_and_managing_streams_for_apache_kafka_on_openshift/using-the-topic-operator-str#doc-wrapper"
```

Comment:
@PaulRMellor, these are the links I mentioned. I think they should be accurate based on the information you've provided.

Reply:
Look Good, Mike